tests/test_routing.py

```diff
@@ -140,3 +140,31 @@ def test_conditional_check_proper_compile(handler):
     router.finalize()
 
     assert router.finalized
+
+
+@pytest.mark.parametrize(
+    "param_name",
+    (
+        "fooBar", "foo_bar",
```

Comment:
Can you add `Foobar` and `foobar1` as tests? In theory any valid python variable should work here.

Reply:
```
([a-zA-Z_][a-zA-Z0-9_]+)
```

Reply:
Add a support `foobar1` and `Foobar`